### PR TITLE
GitHub: Switch to new issue template system

### LINKED ISSUE #3861: Multiple issue templates and template chooser config
Github now supports having several issue templates, using a table at the top to specify a title and description, as well as automatically assigning some tags.
In addition we can supply custom contact links, which will be helpful to avoid password change requests and the like being opened on the tracker

See https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository

### PR DESCRIPTION
**Pull request by skwerlman**
## Related Ticket(s)
- Fixes #3861

## Short roundup of the initial problem
Github has created a new, more flexible issue template system

## What will change with this Pull Request?
- we will use the new system
- hopefully less server support requests

## Screenshots
N/A

---
This PR is a rough draft; please comment if you think something could be done better

### CODE REVIEW

**Comment by skwerlman** [comment]
see https://github.com/skwerlman/cockatrice-pull-3862/issues/new/choose for an example of how the new system looks to users

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Community Discord Server (Login support requests go here!)
```
Comment (tooomm) [inline .github/ISSUE_TEMPLATE/config.yml]: "Community Discord Server (Login support requests go here!)"

Maybe rephrase that to something like
"Discord Community (Support for all server problems)"?

**Comment by skwerlman** [comment]
> We could tweak the labels in the repo and add simple (`Bug` and `Feature Request` ones) to tag both issue types right away and filter for those easily. Further labels with details about OS and level of impact still needs to be applied manually.

This was the idea, but i cant make new labels haha

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (skwerlman) [reply .github/ISSUE_TEMPLATE/config.yml]: i personally think its important to specifically call out login support requests because those are the most common kind we get here, tho i'm ofc fine with going with something else if there's a consensus around it

Comment (tooomm) [reply .github/ISSUE_TEMPLATE/config.yml]: I prefer it to sound more general, and not limited to login issues only.
"Discord Community (Find help with server issues like `Login`)"?
"Discord Community (Support for server issues: Account, Login...)"?
...

**Comment by tooomm** [comment]
>This was the idea, but i cant make new labels haha

I created the labels and added them to the templates.
There are some eye catching emojis now, too.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: false
 contact_links:
   - name: 💬 Community Discord Server (Login support requests go here!)
-    url: https://discord.gg/V6ckZEs
+    url: https://discord.gg/3Z9yzmA
```
Comment (skwerlman) [inline .github/ISSUE_TEMPLATE/config.yml]: why did this need to be changed? it was already an non-expiring, unlimited uses invite to the discord server

**Comment by tooomm** [comment]
What do you think about the current status of this @skwerlman?
Do you want further feedback or changes here?
I like it and hope to see it in use, soon. :)

Only had this additional idea in https://github.com/Cockatrice/Cockatrice/pull/3862#issuecomment-552203993 to mark not yet processed issues.
